src/app/navbar/navbar.component.html

```diff
@@ -41,10 +41,16 @@
           <img class="site-icons-small hidden-sm" src="../assets/svg/main-nav/icons-navigation-forum.svg" alt="stacked pages" /> Forum
         </a>
         <span class="spacer"></span>
-        <a *ngIf="!isLoggedIn" routerLink="/login">
-          <button mat-raised-button color="accent">Login</button>
-        </a>
-        <template [ngTemplateOutlet]="loginButton"></template>
+        <div fxLayoutGap="1rem">
+          <a *ngIf="!isLoggedIn" routerLink="/register" [state]="{ data: 1 }">
+            <button mat-raised-button color="accent">Register</button>
+          </a>
+          <template [ngTemplateOutlet]="loginButton"></template>
+          <a *ngIf="!isLoggedIn" routerLink="/login" [state]="{ data: 0 }">
+            <button mat-raised-button color="accent">Login</button>
+          </a>
+          <template [ngTemplateOutlet]="loginButton"></template>
```

Comment:
Do you need this twice? It's also on line 48.

Reply:
Thank you for catching this as well, I don't need it twice.